v2/apiserver/config.go

```diff
@@ -140,12 +140,28 @@ func sessionsServiceConfig(
 		return config, err
 	}
 	if config.RootUserEnabled {
+		rootUserSessionTTL, err := os.GetRequiredEnvVar("ROOT_USER_SESSION_TTL")
+		if err != nil {
+			return config, err
+		}
+		config.RootUserSessionTTL, err = time.ParseDuration(rootUserSessionTTL)
+		if err != nil {
+			return config, err
+		}
```

Comment:
This can be substantially shortened using `os.GetDurationFromEnvVar`:

```
if config.RootUserSessionTTL, err = os.GetDurationFromEnvVar("ROOT_USER_SESSION_TTL", time.Hour); err != nil {
  return config, err
}
```

Same for user session TTL.

Reply:
Perfect!  Thank you; updated.